staging/src/k8s.io/apiserver/pkg/endpoints/installer.go

```diff
@@ -654,12 +654,13 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 				string(types.JSONPatchType),
 				string(types.MergePatchType),
 				string(types.StrategicMergePatchType),
+				string(types.ApplyPatchType),
```

Comment:
Should this be behind the feature gate? like:
```
if utilfeature.DefaultFeatureGate.Enabled(features.ServersideApply) {
    supportedTypes := appends(supportedTypes, types.ApplyPatchType)
}
```

Reply:
Go for it!

Reply:
Consequence of that (AFAIU):
- documentation and committed swagger file won't have this content-type. 
- apiserver will still generate valid swagger.
- We check that the content-type is valid before doing anything, so that's good enough to prevent people from using that content-type.

I think that's good.